1molehayo/Luminskin
# #BugFix: overflow bug in cart

### What does this MR do?
- fix overflow bug in the cart


### How can it be manually tested?
- launch app
- add plenty products to the cart
- open cart to see changes
